src/components/actionJSForm.js

```diff
@@ -16,26 +16,12 @@
     }
 
     const onActionSuccess = (response) => {
-      /* eslint-disable-next-line */
-      if (response.errors) {
-        const messages = response.errors.flatMap((error) => {
-          if (!error || !error.messages || !error.messages.errors) return [];
-          return error.message.errors.map((inner) => inner.message);
-        });
-
-        B.triggerEvent('onActionError', new Error(messages.join(', ')));
-
-        console.log('Errors: ', messages.join('\n   '));
-        return;
-      }
-
```

Comment:
Was the removal of the error handling removed deliberately? Is it correct that this is handled via the alert component now? As I see there have been some additions regarding error messages there.

Reply:
Yes, the error handling has been moved there. 

Ideally this should be done inside a helper

Reply:
Created issue PAGE-1845 for the helper

Reply:
:ok_hand: